src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js

```diff
@@ -35,16 +38,16 @@ export default class Input extends React.PureComponent<Props> {
     };
 
     handleBlur = () => {
-        const {onFinish} = this.props;
+        const {onBlur} = this.props;
 
-        if (onFinish) {
-            onFinish();
+        if (onBlur) {
```

Comment:
@chirimoya WDYT about this? We need the `Input` component in a non-form environment, therefore we decided to remove the `FieldTypeProps` from this component and introduce a separate field (this is a good idea IMO). What's bothering me a little bit is the renaming of `onFinish` to `onBlur` here. For me it makes totally sense, because the `onBlur` is what you are also used to in HTML. But where do we stop then? Should the `AutoComplete` and `ResourceLocator` component also have an `onBlur` instead of an `onFinish` then?

Reply:
+1 `onBlur` for all components which are close to an `Input` implementation

Reply:
@trickreich Talked to @chirimoya, I'd say we change it also to `onBlur` in the `ResourceLocator` and `AutoComplete` in this PR.

At some point we can also try to do that for all the other components currently implementing the `FieldTypeProps` type (after a short check I've found `FieldBlocks` and `MediaSelection`) and move the `FieldTypeProps` to the form, but I think that is not necessary for this PR.